manager/state/raft/raft.go

```diff
@@ -342,6 +398,9 @@ func (n *Node) JoinAndStart(ctx context.Context) (err error) {
 	}
 
 	// join to existing cluster
+	if n.opts.Addr == "" {
```

Comment:
Then I don't understand how does it work. Isn't it possible to call `JoinAndStart` before `BindRemote` and get `attempted to join raft cluster without knowing own address`?